examples/preprocessing/plot_target_encoder_cross_val.py

```diff
@@ -33,12 +38,16 @@
 n_categories = 100
 
 kbins = KBinsDiscretizer(
-    n_bins=n_categories, encode="ordinal", strategy="uniform", random_state=rng
+    n_bins=n_categories,
+    encode="ordinal",
+    strategy="uniform",
+    random_state=rng,
+    subsample=None,
```

Comment:
Why is disabling subsampling here?

Reply:
Sorry, should have explained. This was just to avoid the FutureWarning:
```
/home/circleci/project/sklearn/preprocessing/_discretization.py:239: FutureWarning:

In version 1.5 onwards, subsample=200_000 will be used by default. Set subsample explicitly to silence this warning in the mean time. Set subsample=None to disable subsampling explicitly.
```
Was not sure of the original intent, I think current default is `None`, so have used this but happy to change to new default 200_000 ?